src/docs/guide/tutorials/your_first_http4k_app/index.md

```diff
@@ -71,10 +71,18 @@ In `/src/main/kotlin/HelloWorldClient.kt` there is an example of an HTTP client
 Try running the client by hitting the little green arrow on **line 11**. You'll see the `Response` printed to the console by the `Filter`, followed by a repeat of the body content, which is printed by **line 18**.
 
 #### Step 6
-Modify the request in `/src/main/kotlin/HelloWorldClient.kt` to point at `http://localhost:9000/pong` instead. Run the client program again and note that a 404 is printed. This happens because we have not bound an `HttpHandler` to that endpoint.
+Modify the request in `/src/test/kotlin/HelloWorldClient.kt` to point at `http://localhost:9000/pong` instead. Run the client program again and note that a 404 is printed. This happens because we have not bound an `HttpHandler` to that endpoint.
+
+To bind an `HttpHandler` to the `pong` endpoint, modify `/src/main/kotlin/HelloWorld.kt` by adding lines 18-20 below:
+
+<img class="blogImage" src="step6.png" alt="client code"/>
```

Comment:
Oh I see, this is a PR into your fork of `http4k` rather than your specific project? Fleshing out their library documentation is a great idea. You might even be able to submit it back to the original project, which would be an additional nice output for the project.

A little confused by changing the path from `src/main` to `src/test`. Typically, you also set up a `test/` directory in the project root and you try to mirror the `src/` file tree structure there, which would mean you would end up having `test/test`...

Reply:
Oh, belay my comment about the paths. Sorry, I should have checked their project out first! Confusing that they have `src/main` and `src/test` instead, but I guess Kotlin deviates from Java convention there.

Reply:
Now that I actually went and read the tutorial as I probably should have done from the beginning, I think it would be a nice touch if you actually put this after *Step 7* and had them write a test for the `/pong` endpoint first, then implement it.